createdisk.sh

```diff
@@ -33,12 +33,17 @@ function create_disk_image {
 # CRC_VM_NAME: short VM name to use in crc_libvirt.sh
 # VM_PREFIX: full VM name with the random string generated by openshift-installer
 CRC_VM_NAME=${CRC_VM_NAME:-crc}
+VIRT_SPARSIFY=${VIRT_SPARSIFY:virt-sparsify}
 
 if [[ $# -ne 1 ]]; then
    echo "You need to provide the running cluster directory to copy kubeconfig"
    exit 1
 fi
 
+if ! which ${VIRT_SPARSIFY}; then
+    sudo yum install /usr/bin/virt-sparsify
```

Comment:
Does this install required package? Also you need to use `-y` flag so there is no human intervention. Can you add same for `qemu-img` since this is also used here?

Reply:
Yes this will do the right thing. I'll add -y and qemu-img.